source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -551,6 +631,21 @@ export default class WalletRestoreDialog extends Component<Props, State> {
     return this.state.walletType === WALLET_RESTORE_TYPES.LEGACY;
   }
 
+  isYoroiLegacy() {
+    return this.state.walletType === WALLET_RESTORE_TYPES.YOROI_LEGACY;
+  }
+
+  isYoroi() {
```

Comment:
@DeeJayElly 
Use `isYoroiLegacy()` and `isYoroiRegular()`

Reply:
Fixed